src/human-readable/runtime/cache.ts

```diff
@@ -0,0 +1,55 @@
+import type { AbiParameter } from '../../abi'
+
+export function createParameterCache() {
```

Comment:
Renamed. Let's also add a comment about why certain parameters are seeded, max number of seeded values, etc.

```suggestion
export function createAbiParameterCache() {
```

Reply:
Okay that sounds good. Will update it